Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -294,10 +290,3 @@
 "captcha-reload" = "Title for button used to request another CAPTCHA to solve";
 "captcha-prompt" = "Text prompting user to solve a CAPTCHA";
 "mwk-empty-title-error" = "Error description for when an operation couldn't be performed since no title was specified.";
-"home-title" = "Title for home interface.";
```

Comment:
@montehurd @bgerstle I just tried again (and forced push). No luck. Do you guys normally use this update-qqq script? I've just been running the TWN scripts (repoupdate, repoexport, ...) and don't know anything about it.